sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/translation/SqlOperators.java

```diff
@@ -132,6 +134,8 @@
   public static final SqlOperator DATE_OP =
       createUdfOperator("DATE", BeamBuiltinMethods.DATE_METHOD);
 
+  // TODO: Fix Later
```

Comment:
Same here, link to an issue so someone who finds the TODO can see if anyone else is working on it or if there is some history.